### ISSUE TITLE
[Windows] Non-default Documents location

### ISSUE THREAD

**Issue by eivindveg** [eivindveg/HotSUploader#11]
It's increasingly normal to move the location of "Documents" to avoid size creep on SSDs. This results in Documents not being located in "user.home". A workaround is properly symlinking the Documents folder so that it also appears to be located in "user.home". However, this is beyond the scope of expectations for a normal user, and a more robust lookup should be implemented in the future.

If you're affected by this bug, please look at [this link](http://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/)

**Comment by christianhood** [comment]
Would it not be better to store the upload statuses into the appdata folder on Windows instead? After doing a little research I believe you can get this with `System.getenv("APPDATA");`.

**Comment by eivindveg** [comment]
That would require a migration or invalidation, but otherwise would be equivalent to the OSX route we employ.

**Comment by eivindveg** [comment]
Will use appdata in the future. First we need to get Windows users on to a migrateable build base

**Comment by CaptinKraken** [comment]
For those wanting to use this uploader, with your Documents folder somewhere else besides your C:\ drive on Windows.

You'll just need to create a symlink/Directory Junction of your Heroes of the Storm folder, into your C:\ Documents folder.

In cmd prompt w/ administrative privileges:
 ```
mklink /j "C:\Users\CaptinKraken\Documents\Heroes of the Storm" "e:\Documents\Heroes of the Storm"
```

This will create a link your C:\ Documents folder, which this program needs, to your actual Documents Folder. No files will be duplicated.

**Comment by britishben** [comment]
It doesn't look like you're using the %userhome% for anything, just the "My Documents". Unfortunately, Java doesn't really provide an easy way to get that. Would something like [this](http://stackoverflow.com/a/9677746) work better?

**Comment by eivindveg** [comment]
Will be transitioning to that or something else that ends up working as soon as I can find some time to write a proper migration. Don't want to just orphan the files, or force users to reupload. Currently somewhat busy on a project for my school's student society, so I'm slower than usual.

**Comment by eivindveg** [comment]
Fixed in 7800817b8218dd8fe8ff45b7f7a9296009c6e602
